dragonchain/webserver/lib/dragonnet.py

```diff
@@ -80,13 +87,51 @@ def process_receipt_v1(block_dto: Dict[str, Any]) -> None:
                 _log.exception("matchmaking add_receipt failed!")
             # Update the broadcast system about this receipt
             broadcast_functions.set_receieved_verification_for_block_from_chain_sync(l1_block_id, level_received_from, block_model.dc_id)
+            # Send verifications to any configured urls
+            send_verification_notifications(level_received_from, block_model)
         else:
             _log.warning(
                 f"Chain {block_model.dc_id} (level {level_received_from}) returned a receipt that wasn't expected (possibly expired?) for block {l1_block_id}. Rejecting receipt"  # noqa: B950
             )
             raise exceptions.NotAcceptingVerifications(f"Not accepting verifications for block {l1_block_id} from {block_model.dc_id}")
 
 
+def get_notification_urls(key: str) -> set:
+    urls = None
+    try:
+        urls = VERIFICATION_NOTIFICATION[key]
+    except KeyError:
+        pass
+    if urls is None:
+        urls = []
+    return set(urls)
+
+
+def sign(message: str) -> Tuple[str, Optional[str]]:
+    dc_keys = keys.DCKeys()
+    signature = dc_keys.make_signature(message.encode("utf8"), crypto.SupportedHashes.sha256)
+    return (signature, dc_keys.pub)
+
+
+def send_verification_notifications(level: int, block_model: "model.BlockModel"):
+    try:
+        if VERIFICATION_NOTIFICATION:
+            string_json = json.dumps(block_model.export_as_at_rest())
```

Comment:
add separators tuple as per every other json.dumps in the project

Reply:
Also note the `.export_as_at_rest()` is a potentially changing schema as dragonchain versions change.

If you want to ensure this is a static schema, then you'll have to craft a custom output dictionary, like webserver handlers

Reply:
spoke IRL. We are cool with this